framework/CodeInterfaces/MAAP5/MAAP5Interface.py

```diff
@@ -643,7 +643,7 @@ def stopSimulation(self,currentInputFiles, Kwargs):
 ####################
         raise IOError('All TIMER must be considered for the first simulation') #in the original input file all the timer must be mentioned
     else:
-      #Kwargs['parentID'] != 'root'
+      #Kwargs['RAVEN_parentID'] != 'None'
```

Comment:
should this be removed?

Reply:
I want to keep this because the Interface has been developed by external users and they will need to know that the parentID keyword changed